src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,74 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(name = "movie_id")
+    private Movie movie;
+
+    @ManyToOne
+    private CinemaHall cinemaHall;
```

Comment:
Consider adding a `@JoinColumn` annotation for the `cinemaHall` field to specify the foreign key column name in the `movie_sessions` table. This will ensure that the mapping is explicit and clear.